packages/commonwealth/client/styles/modals/linked_thread_modal.scss

```diff
@@ -2,14 +2,16 @@
 
 .LinkedThreadModal {
   width: 504px;
+  max-height: 84vh;
+  max-width: 100vw;
```

Comment:
again, these two lines are repeated like 20 times. And I am not sure why it is exactly 84vh? Should't we have fixed height of the modals?

Reply:
It was defined on first spec on the ticket. Moved to parent CWModal component.